src/sources/forus-webshop/pug/tpl/pages/provider.pug

```diff
@@ -18,7 +18,7 @@ block content
                 .pane-head Aanbieder informatie
                 .pane-section: .provider-details
                     .provider-description
-                        img(ng-src="{{ $ctrl.provider.logo.sizes.thumbnail || '/assets/img/placeholders/organization-thumbnail.png' }}").provider-logo
+                        img(ng-src="{{ $ctrl.provider.logo.sizes.thumbnail || '/assets/img/placeholders/organization-thumbnail.png' }}" alt="Aanbieder logo").provider-logo
```

Comment:
As it is aswell sometimes a thumbnail the alt text is not correct. I believe this image is purely decorative; maybe just add the alt text and leave it empty